DataConnectors/McAfeeePO/Connector_McAfee_ePO.json

```diff
@@ -0,0 +1,124 @@
+{
+    "id": "McAfeeePO",
+    "title": "McAfee ePolicy Orchestrator (ePO)",
+    "publisher": "McAfee",
+    "descriptionMarkdown": "The McAfee ePolicy Orchestrator data connector provides the capability to ingest [McAfee ePO](https://www.mcafee.com/enterprise/en-us/products/epolicy-orchestrator.html) events into Azure Sentinel through the syslog. Refer to [documentation](https://docs.mcafee.com/bundle/epolicy-orchestrator-landing/page/GUID-0C40020F-5B7F-4549-B9CC-0E017BC8797F.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **McAfeeEPOEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-McAfeeePO-parser)\nNote: This data connector has been developed using McAfee ePolicy Orchestrator 5.10.x (Syslog)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "McAfee ePO Events",
+            "baseQuery": "McAfeeEPOEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Sources",
+            "query": "McAfeeEPOEvent\n | summarize count() by DvcHostname\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog(McAfee ePO)",
```

Comment:
DATATYPE_NAME does not have spaces, Can you please remove space in "McAfee ePO" and make it "McAfeeePO".